src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,23 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private final List<Bulldozer> bulldozers = new ArrayList<>();
+
+    {
+        bulldozers.add(new Bulldozer("John Deere", "green"));
+        bulldozers.add(new Bulldozer("CAT", "yellow"));
+        bulldozers.add(new Bulldozer("New Holland", "blue"));
+    }
+
+    @Override
+    public List<Bulldozer> get() {
+        return bulldozers;
```

Comment:
create and return new list every time

Reply:
not fixed...
I asked to create and return a new list every time. In this method. Now you create one list once and return it every time when the method is called.